contents/tutorials/bounce-rate.md

```diff
@@ -0,0 +1,57 @@
+---
+title:  How to calculate bounce rate
+date: 2023-06-21
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-15.png
+tags: ['insights', 'hogql']
+---
+
+Bounce rate is the percentage of users who leave your page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors.  
+
+This tutorial shows you how to calculate bounce rate in PostHog. To get started, you need to install the [snippet](/docs/getting-started/install?tab=snippet) or [JavaScript SDK](/docs/libraries/js) and enable "record user sessions"  in [project settings](https://app.posthog.com/project/settings).
+
+## Calculating bounce rate with SQL insights
+
+To calculate bounce rate, we need data from `raw_session_replay_events`, which we can access with [SQL insights](/docs/product-analytics/hogql#sql-insights). To create a new SQL insight, go to the insight tab, click [new insight](https://app.posthog.com/insights/new), then go to the SQL tab. This is where we write our SQL statement.
+
+We count a bounce as a session where the user is active for less than 3 seconds. To do this in SQL, we get a count of sessions (using `session_id`) where `active_milliseconds` is less than `3000` and divide by the total session count, then multiply by `100`. Together, this looks like this:
```

Comment:
> We count a bounce as a session where the user is active for less than 3 seconds

Hmm this is different from the standard definition of "user landed on a page and did nothing else". We should stick to this definition (I'm not familiar with the "less than 3 seconds" one)

<img width="808" alt="Screenshot 2023-06-22 at 7 41 18 AM" src="https://github.com/PostHog/posthog.com/assets/7090054/5e1c4c6a-8d34-43e6-8dd4-0cd8da7a0800">

Reply:
The traditional "single page visits divided by total visits" isn't possible. Updated now to make clearer we are using similar to GA4 definition.

Could also do no clicks/keypresses as main definition. That would match "land on page and leave without performing a specific action".